graphql/tests/version/buildVariantStats/data.json

```diff
@@ -309,10 +314,11 @@
       "version": "5e4ff3abe3c3317e352062e4",
       "gitspec": "5e823e1f28baeaa22ae00823d83e03082cd148ab",
       "build_variant": "ubuntu1604",
+      "build_variant_display_name": "Ubuntu 16.04",
       "display_name": "test-agent",
       "r": "github_pull_request",
       "status": "undispatched",
       "build_id": "evergreen_ubuntu1604_patch_5e823e1f28baeaa22ae00823d83e03082cd148ab_5e4ff3abe3c3317e352062e4_20_02_21_15_13_48"
     }
   ]
-}
+}
```

Comment:
This is a mega nit, but if your editor auto-trims newlines at the end of the file, it's considered better practice to configure it to [leave them in](https://stackoverflow.com/a/5813359) because they're used as the EOF indicator in various applications.